runtime/include/chpl-tasks.h

```diff
@@ -193,7 +193,7 @@ void chpl_task_yield(void);
 //
 // Suspend.
 //
-void chpl_task_sleep(int);
+void chpl_task_sleep(int, double);
```

Comment:
In the interest of simplicity I'm inclined to say let's instead have the runtime chpl_task_sleep() continue to take just a single argument specifying the number of seconds to sleep, but make it a double so that it can express fractions of a second also. This would simplify things by giving us just one copy (in Time.chpl) of the code that converts some number of units into seconds.